### ISSUE TITLE
Cli - When log in is attempted ,config generated from previous login attempts is being used.

### ISSUE THREAD

**Issue by sangeethah** [rancher/rancher#11470]
rancher cli  version - v1.0.0-alpha5

I had logged in as admin and made a selection for default project.
Logging  in as another user (who did not have access to this default project) results in following error:
```
Sangeethas-MBP:rancher-v1.0.0-alpha5 sangeethahariharan1$ ./rancher login -t token-**** --cacert ~/ca-cert --name test1 https://****/v3
FATA[0001] Bad response statusCode [403]. Status [403 Forbidden]. Body: [baseType=error, code=Forbidden, message=clusters.management.cattle.io "cluster-f2kr2" is forbidden: User "user-smr6g" cannot get clusters.management.cattle.io at the cluster scope] from [https://****/v3/clusters/cluster-f2kr2] 
Sangeethas-MBP:rancher-v1.0.0-alpha5 sangeethahariharan1$ 
```

**Comment by deniseschannon** [comment]
Available with v1.0.0-alpha7

**Comment by sangeethah** [comment]
Tested with rancher cli version - v1.0.0-alpha7

This issue is not seen anymore.
